lib/src/components/NxToast/NxToastContainer.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useRef, useEffect } from 'react';
+import NxToastContainerContext from './contexts';
+import {reject, last} from 'ramda';
+
+import { NxToastContainerProps, nxToastContainerPropTypes } from './types';
+
+import './NxToast.scss';
+
+export { Props } from './types';
+
+const NxToastContainer = (props: NxToastContainerProps) => {
+  const { children } = props;
+
+  const ref = useRef<HTMLDivElement | null>(null);
+  const activeElementNotToast = useRef<HTMLElement | null>(null);
```

Comment:
This could use a brief comment explaining what it is

Reply:
> This could use a brief comment explaining what it is

Explaining comment added here: [ab10adc](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/ab10adc151c7e3d33b7eeb0fc70172eeed7c6975)